client/src/app/user/shared/register.reducer.ts

```diff
@@ -0,0 +1,18 @@
+import { Action } from '@ngrx/store';
+
+
+export interface State {
```

Comment:
__[TSLint]__ An empty interface is equivalent to `{}`. (no-empty-interface)

<sub>posted by Sider</sub>